### PR TITLE
docs(contributing): Create contributing file

### LINKED ISSUE #7: update contributing
move contributing to its own file & explain our labels attached to issues

### PR DESCRIPTION
**Pull request by GabrielShaad**
fix #7

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# How to Contribute
+
+Thank you for taking your time to consider contributing to the repository!
+
+To get started, follow the instructions in this document.
+
+## Report a bug fix
+
+To report a bug fix, follow the issue template found when openning a new issue, if possible and available attach a [codesandbox](https://codesandbox.io/) or [jsBin](https://codesandbox.io/) with the error being reported.
```
Comment (santospatrick) [inline CONTRIBUTING.md]: jsbin link should be https://jsbin.com/?js,output

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+This will help you understand how you should develop your contribution to the repository:
+
+![Repository Workflow](images/workflow.png)
+
+## Install Dependencies
+
+After getting the repository locally in your machine, you have to install all the dependencies in order to run and start contributing.
+
+To install everything you need, go to the folder you cloned the repository and run: _yarn install_
```
Comment (santospatrick) [inline CONTRIBUTING.md]: show the command like this
```bash
yarn install
```
its better then "_yarn install_" in a big doc

File: CONTRIBUTING.md (hunk 1 above)
Comment (santospatrick) [inline CONTRIBUTING.md]: link to jsbin should be https://jsbin.com/?js,output

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+| question         | Indicates that an issue or pull request needs more information     |
+| wontfix          | Indicates that work won't continue on an issue or pull request     |
+
+## Fork or Clone It
+
+In order to start colaborating to our project, you have to **FORK** this repository if you are a contributor, or **CLONE** it if you are a not a contributor.
+
+## Development Workflow
+
+This will help you understand how you should develop your contribution to the repository:
```
Comment (santospatrick) [inline CONTRIBUTING.md]: get the link for slides of this workflow, even with the image it is complicated to understand each step

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+Add the files you want to commit, and type the following command: _yarn run commit_
+
+Carefully read and follow the instructions to finish your commit.
+
+After commiting, push to the remote branch you previously created: _git push origin my-new-branch_
+
+## Pull Request
+
+Last but not least, verify if your chnages were pushed to your remote branch.
```
Comment (santospatrick) [inline CONTRIBUTING.md]: typo in "changes"

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+After getting the repository locally in your machine, you have to install all the dependencies in order to run and start contributing.
+
+To install everything you need, go to the folder you cloned the repository and run: _yarn install_
+
+## Testing your Component
+
+If your contribution is a new component, or a modification in an existing one, you might test it's final UI version with Storybook.
+
+To test the component, type this command in the project's folder location: _yarn run storybook_
```
Comment (santospatrick) [inline CONTRIBUTING.md]: show the command like this

```bash
yarn run storybook
```

its better then "_yarn run storybook_" in a big doc

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+After running the command, go to localhost:6006 in your favorite browser and look for the component you created or changed
```
Comment (santospatrick) [inline CONTRIBUTING.md]: maybe make "localhost:6006" bold, like this: **localhost:6006**

**Comment by AgrotisArquitetura** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please assign someone to merge this PR, and optionally include people who should review.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Pull request merged**